cpp/benchmarks/CMakeLists.txt

```diff
@@ -61,7 +76,11 @@ function(ConfigureNVBench CMAKE_BENCH_NAME)
     ${CMAKE_BENCH_NAME}
     PROPERTIES RUNTIME_OUTPUT_DIRECTORY "$<BUILD_INTERFACE:${CUSPATIAL_BINARY_DIR}/benchmarks>"
                INSTALL_RPATH "\$ORIGIN/../../../lib"
-  )
+               CXX_STANDARD 17
+               CXX_STANDARD_REQUIRED ON
+               CUDA_STANDARD 17
+               CUDA_STANDARD_REQUIRED ON
+)
```

Comment:
`target_compile_features(... cxx_std_17 cuda_std_17)` doesn't actually set the `-std=c++17` argument, we have to define the `{CXX,CUDA}_STANDARD` properties for that.